### ISSUE TITLE
Entries sometimes do not have statuses

### ISSUE THREAD

**Issue by Samasaur1** [Samasaur1/rssbot#3]
I don't really understand why this would be, but I got the following error:

```
Unexpected err=AttributeError("object has no attribute 'status'"), type(err)=<class 'AttributeError'>
```

and I narrowed it down to this line:

https://github.com/Samasaur1/rssbot/blob/cc21b0699b8248c676f0244532863e103ff4cd85/rssbot.py#L86

**Comment by Samasaur1** [comment]
log:
```
Jul 19 00:16:23 blinky rssbot-start[3078]: verbose: in new_entries
Jul 19 00:16:23 blinky rssbot-start[3078]: verbose: previous entry InternetUnexplorer created a branch auto-update/flake-inputs in InternetUnexplorer/nixpkgs-overlay
Jul 19 00:16:23 blinky rssbot-start[3078]: verbose: etag/modified
Jul 19 00:16:23 blinky rssbot-start[3078]: ERR: no status attribute
Jul 19 00:16:23 blinky rssbot-start[3078]: https://github.com/InternetUnexplorer.atom
Jul 19 00:16:23 blinky rssbot-start[3078]: <__main__.Entry object at 0x7fdf6d25bbb0>
Jul 19 00:16:23 blinky rssbot-start[3078]: W/"16c112a4d92fa61607ac7ebc0e015cfe"
Jul 19 00:16:23 blinky rssbot-start[3078]: None
Jul 19 00:16:23 blinky rssbot-start[3078]: {'bozo': True, 'entries': [], 'feed': {}, 'headers': {}, 'bozo_exception': URLError(gaierror(-2, 'Name or service not known'))}
Jul 19 00:16:23 blinky rssbot-start[3078]: Unexpected err=AttributeError("object has no attribute 'status'"), type(err)=<class 'AttributeError'>
```

**Comment by Samasaur1** [comment]
Addressed in 76682286e07b1f4f7ada97a5da2bd545fb300ac1 (and previous commits)
